Reply/app/src/main/java/com/example/reply/ui/theme/Color.kt

```diff
@@ -2,10 +2,60 @@ package com.example.reply.ui.theme
 
 import androidx.compose.ui.graphics.Color
 
-val Purple80 = Color(0xFFD0BCFF)
-val PurpleGrey80 = Color(0xFFCCC2DC)
-val Pink80 = Color(0xFFEFB8C8)
+// Generate them via theme builder
+// https://material-foundation.github.io/material-theme-builder/#/custom
 
-val Purple40 = Color(0xFF6650a4)
-val PurpleGrey40 = Color(0xFF625b71)
-val Pink40 = Color(0xFF7D5260)
\ No newline at end of file
+val reply_light_primary= Color(0xFF825500)
```

Comment:
I filed a bug for the theme builder about this, but... We should change these from snake_case to camelCase

Reply:
Yes I noticed I will raise it to material team as well 👍🏽 Please share link of buganizer if you have.